core/domain/user_jobs_one_off_test.py

```diff
@@ -222,6 +223,84 @@ def test_multiple_users_case(self):
         self.assertEqual(output['2'], 2)
         self.assertEqual(output['1'], 1)
 
+class UserBioLengthOneOffJobTests(test_utils.GenericTestBase):
+    """Tests for the one-off username length distribution job."""
+    USER_A_EMAIL = 'a@example.com'
+    USER_A_USERNAME = 'a'
+    USER_A_BIO = 'I am less than 500'
+    USER_B_EMAIL = 'b@example.com'
+    USER_B_USERNAME = 'b'
+    USER_B_BIO = 'Long Bio' *100
+    USER_C_EMAIL = 'c@example.com'
+    USER_C_USERNAME = 'cnone'
+    USER_C_BIO = 'Same Bio' *100
+
+    def _run_one_off_job(self):
+        """Runs the one-off MapReduce job."""
+        job_id = (
+            user_jobs_one_off.UserBioLengthOneOffJob.create_new())
+        user_jobs_one_off.UserBioLengthOneOffJob.enqueue(job_id)
+        self.assertEqual(
+            self.count_jobs_in_taskqueue(
+                queue_name=taskqueue_services.QUEUE_NAME_DEFAULT),
+            1)
+        self.process_and_flush_pending_tasks()
+
+        stringified_output = (
+            user_jobs_one_off.UserBioLengthOneOffJob.get_output(
+                job_id))
+        output = {}
+        #for loop is needed to void `Index out of range` error
+        for stringified_output[0] in stringified_output:
```

Comment:
In here I need to explain - 
Stringified_output is nothing but the output converted to a string. Right? Alike -- 
If output is ` [u'800', [u'cnone', u'b']` then stringifiyed output will be 
`stringified_output= [u"[u'800', [u'cnone', u'b']]"]`